### PR TITLE
Updates codeception dependency to 3.0

### PR DESCRIPTION
**Pull request by patrickvale**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "description": "Generate Gherkin steps from Codeception Modules.",
   "type": "library",
   "require": {
-    "codeception/codeception": "^2.2"
+    "codeception/codeception": "^3.0"
```
Comment (lucatume) [inline composer.json]: I use this in projects using `2.*`; what about `"codeception/codeception": ">=2.2"?

**Comment by lucatume** [comment]
Thanks for the PR, could you check if my proposed change would make sense?

**Comment by patrickvale** [comment]
Hi, sorry for the delay in reviewing this.

Yes - that makes perfect sense. I was using it on a new project so just thinking about 3.x support. Having backwards compatibility is always a great win!

Thanks for this project - it's really helpful.

**Comment by lucatume** [comment]
Thanks for your PR. I’ve not seen many usages of this in the wild: would you mind sharing how and where you use it? 

Mine is just curiosity, if time or copyright does not allow it, no worries.

**Comment by ivanrey** [comment]
Packagist only has this update in dev-master, could you make a release to see it without requiring "dev-master" in composer.json?

**Comment by lucatume** [comment]
> Packagist only has this update in dev-master, could you make a release to see it without requiring "dev-master" in composer.json?

Right, thanks for bringing this up, done.

**Pull request merged**